index.html

```diff
@@ -176,6 +176,22 @@ <h2 class="headline-section__sub-title">
 		</ul>
 			</div>
 		</section>
+		<section class="contact-form-section">
+			<div class="contact-form__container">
+				<div class="outro__box">
+					<h2 class="healine-section__title white">Contact me</h2>
+					<p class="headline-section__desc white">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+				</div>
+				<form action="https://formspree.io/f/" method="post">
+                    <label for="text"><input  type="text" required placeholder="Full Name" maxlength="30" id="text"></label>
+                    <label for="email"><input name="Email" id="email" type="email" required maxlength="30" placeholder="Your email"></label>
+					<textarea cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+                    <div class="btn__submit">
+						<button type="submit">Get in touch</button>
```

Comment:
- [ ] **Required:** Please verify that the form is submitted properly. When I tried to submit it, I got the error below.
![image](https://user-images.githubusercontent.com/27709832/149174268-656ea43d-4bc6-48a3-948d-ac82c582a16e.png)

Reply:
I have just updated the form action